scripts/pico_project.py

```diff
@@ -356,23 +356,17 @@ def cmakeToolchainPath(toolchainVersion):
 def cmakeToolsPath(sdkVersion):
     return f"${{USERHOME}}{relativeToolsPath(sdkVersion)}"
 
-def propertiesSdkPath(sdkVersion, force_windows=False, force_non_windows=False):
-    if (isWindows or force_windows) and not force_non_windows:
-        return f"${{env:USERPROFILE}}{relativeSDKPath(sdkVersion)}"
-    else:
-        return f"${{env:HOME}}{relativeSDKPath(sdkVersion)}"
+def propertiesSdkPath(sdkVersion):
+    return f"${{userHome}}{relativeSDKPath(sdkVersion)}"
 
 def codeSdkPath(sdkVersion):
     return f"${{userHome}}{relativeSDKPath(sdkVersion)}"
 
 def codeOpenOCDPath(openocdVersion):
     return f"${{userHome}}{relativeOpenOCDPath(openocdVersion)}"
 
-def propertiesToolchainPath(toolchainVersion, force_windows=False, force_non_windows=False):
-    if (isWindows or force_windows) and not force_non_windows:
-        return f"${{env:USERPROFILE}}{relativeToolchainPath(toolchainVersion)}"
-    else:
-        return f"${{env:HOME}}{relativeToolchainPath(toolchainVersion)}"
+def propertiesToolchainPath(toolchainVersion):
+    f"${{userHome}}{relativeToolchainPath(toolchainVersion)}"
```

Comment:
This needs a return
```suggestion
    return f"${{userHome}}{relativeToolchainPath(toolchainVersion)}"
```

Reply:
Actually, it might be simpler to just remove propertiesXXXPath and use codeXXXPath instead, as there's no difference between them now?

Reply:
Right, I also thought about this.

Will be implemented...